src/views/Insights/index.vue

```diff
@@ -1,29 +1,29 @@
 <template>
   <div class="wpp_insights">
     <!-- Breadcrumb -->
-    <div class="wpp_insights__breadcrumb">
+    <div class="wpp_insights__breadcrumb" v-if="!hash">
       <unnnic-breadcrumb
         :crumbs="[
-          { name: 'Meus aplicativos', path: '/apps' },
-          { name: 'Insights', path: '/' },
+          { name: $t('apps.nav.my_apps'), path: '/apps/my', meta: 'Meus aplicativos' },
+          { name: 'Insights', path: '/apps/insights', meta: 'Insights' },
         ]"
+        @crumbClick="redirectTo"
       />
     </div>
     <!-- Header -->
-    <div class="wpp_insights__header">
+    <div class="wpp_insights__header" v-if="!hash">
       <div class="wpp_insights__header__logo">
-        <unnnic-avatar-icon icon="messaging-whatsapp-1" scheme="feedback-green" />
+        <img src="../../assets/svgs/whatsapp-avatar.svg" alt="" />
```

Comment:
Asset import should not use a relative import, I believe that `assets/svgs/whatsapp-avatar.svg` will work.